CI/PSI/local_jenkins.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
```

Comment:
@stephan271 I think we can get rid of this file. Do you agree?

Reply:
yes